### PR TITLE
Update 404.html

### LINKED ISSUE #150: Better 404 Page
If anyone has any ideas for a Buddhism-themed 404 page, have at it! 😁

### PR DESCRIPTION
**Pull request by StrixV54**
fixes  #150

### CODE REVIEW

**Review by buddhist-uni — COMMENTED** [review]
Thanks for taking an initial stab at this. Is this really what you intended it to look like?

screenshot

**Comment by StrixV54** [comment]
Yes around something like this, I can fix it more.
Can you please tell me how do I run this, I have never used ruby. Thanks.

**Comment by buddhist-uni** [comment]
Install [Node](https://nodejs.org/en/download/package-manager/), [Ruby](https://www.ruby-lang.org/en/documentation/installation/), and [Bundler](https://bundler.io/).
Then run:

```bash
git clone https://github.com/buddhist-uni/buddhist-uni.github.io.git
cd buddhist-uni.github.io
npm ci
bash scripts/install-deps.bash
bundle install
bundle exec jekyll serve -tI
```

**Review by buddhist-uni — CHANGES_REQUESTED** [review]
In general, care needs to be taken that text overlaid on top of an image is still legible. This can be done many ways: choosing a simple background image, adding text-shadow, blurring the background, adding a semi-transparent overlay between the two layers, etc.  Also, any design should work well at all screen widths between ~300px and ~3000px. This means loading different resolution photos at different widths and making sure the layout adapts flexibly to changing widths.

File: 404.html [hunk 1]
```diff
@@ -6,21 +6,45 @@
 
 <style type="text/css" media="screen">
   .container {
-    margin: 10px auto;
-    max-width: 600px;
+    position: absolute;
+    margin: 0;
+    padding: 0;
+    background-image: url("https://images.unsplash.com/photo-1622354688049-e1a21bcb88bc?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1500&q=80");
```
[the remark this answers is not in the document]
Comment (StrixV54) [reply 404.html:27]: welcome

**Comment by StrixV54** [comment]
Does it looks okay, please change the main.page-context and #main.wrapper for error then it will look good.

**Comment by buddhist-uni** [comment]
I'm sorry, I don't follow what you mean... what error?  It looks much better now though, thanks! :smiley: Should I just merge this, then? Or does this still need some work?

**Comment by StrixV54** [comment]
Can you share a pic  how it looks on your side now? @buddhist-uni

**Comment by buddhist-uni** [comment]
Screenshot from 2023-09-23 18-53-15

**Comment by StrixV54** [comment]
Cool, yes , just wanted to make sure it looks gud on your side too. You can merge and close this PR now.

**Comment by buddhist-uni** [comment]
Thanks for your contribution, @StrixV54 😁

**Pull request merged**